### ISSUE TITLE
When selecting text and clicking the italic or bold button remove potential blank character at the end

### ISSUE THREAD

**Issue by santilland** [EOX-A/EOxElements#1429]
When selecting text which has a blank space at the end adding italic/bold will not display correctly in the preview, as the blank space will "break" the preview.
I think this helps with usability, but it might be not reasonable thing to do, depending on what others think we can close this issue.

**Comment by silvester-pari** [comment]
You're right, this is something that e.g. GitHub does nicely, if inside an issue comment it removes the space automatically. Seeing that this is something that would need to be done inside the EasyMDE editor and there is no easy way to "hook into" the italic/bold toggle, I guess we'll have to live with it (until we replace the EasyMDE editor for something else).
